attest/eventlog.go

```diff
@@ -77,6 +88,9 @@ type EventLog struct {
 func (e *EventLog) Verify(pcrs []PCR) ([]Event, error) {
 	events, err := replayEvents(e.rawEvents, pcrs)
 	if err != nil {
+		if _, isReplayErr := err.(ReplayError); isReplayErr {
+			return nil, err
+		}
 		return nil, fmt.Errorf("pcrs failed to replay: %v", err)
```

Comment:
Use error wrapping? https://golang.org/pkg/errors/#pkg-overview

Reply:
Do we want to require all our users to have Go 1.13? Seems a little early?

Reply:
Yeah you're probably right.